RFC.doku

```diff
@@ -16,26 +18,30 @@ A deprecated feature may throw an exception before major release if the security
 
 ====== Normalization of the deprecation message ======
 
-The deprecation message must include the version of PHP in which the feature will be removed. 
+The deprecation message must include the version of PHP in which the feature will be removed.
+
+===== Defining replacements =====
+
+Active php users — taking care of php evolutions — know which features have to be used instead of the old ones: mysqli vs mysql, preg vs ereg, etc.
 
-===== Define replacements =====
+The deprecated message for mysql is very clear:
+Deprecated: mysql_connect(): The mysql extension is deprecated and will be removed in the future: use mysqli or PDO instead.
+But for ereg it's a little less obvious:
+Deprecated: Function ereg() is deprecated.
 
-Active php users - who take care of evolutions of php - knows which features have to be used in replacement for the old one : mysqli vs mysql, preg vs ereg, etc.
-The deprecated message for mysql is very clear: Deprecated: mysql_connect(): The mysql extension is deprecated and will be removed in the future: use mysqli or PDO instead. 
-But for ereg it's a little less obvious: Deprecated: Function ereg() is deprecated. If the standard deprecated message always include the replacement feature, 
-it will be easier to move forward to new versions of PHP.
+If the standard deprecated message always includes the replacing feature, it will be easier to move forward to new versions of PHP.
 
 
 ====== Exemple of normalized deprecated error ======
 
 
-"Deprecated: \[Feature name] : The feature \[feature name] is deprecated and will be removed in \[PHP Version]. Use \[replacement feature] instead."
```

Comment:
There's no space before colon in english (valid for other correction in the document).